docs/user-guide/persistent-volumes/index.md

```diff
@@ -191,6 +230,41 @@ Claims can specify a [label selector](/docs/user-guide/labels/#label-selectors)
 
 All of the requirements, from both `matchLabels` and `matchExpressions` are ANDed together – they must all be satisfied in order to match.
 
+### Class
+
+A claim can request a particular class by specifying the name of a
+`StorageClass`using the annotation `volume.beta.kubernetes.io/storage-class`.
+Only PVs of the requested class, ones with the same annotation as the PVC, can
+be bound to the PVC.
+
+PVCs don't necessarily have to request a class. A PVC with its annotation set
+equal to `""` is always interpreted to be requesting a PV with no class, so it
+can only be bound to PVs with no class (no annotation or one set equal to
+`""`). A PVC with no annotation is not quite the same and is treated differently
+by the cluster depending on whether the
+[`SimpleDefaultStorageClassForPVC` admission controller](docs/admin/admission-controllers/#simpledefaultstorageclassforpvc)
```

Comment:
it's now `DefaultStorageClass`. And it's not "admission controller", but officially it is "admission plugin"